packages/hardhat-core/src/internal/hardhat-network/provider/vm/exit.ts

```diff
@@ -34,7 +34,7 @@ export class Exit {
       case 0x51:
       case 0x53:
         return new Exit(ExitCode.INVALID_OPCODE);
-      case 0x65:
+      case 0x64:
```

Comment:
Is this correct? [`Return.CreateContractLimit`](https://github.com/bluealloy/revm/blob/90fe01e703dc4c91c3526268bd314745fe82545a/crates/revm/src/instructions.rs#L70) is 0x65, unless I'm missing something

Reply:
![image](https://user-images.githubusercontent.com/6917585/207946910-22295d1c-19c3-427d-a885-e81df26d7218.png)

Reply:
I think it changed recently: https://github.com/bluealloy/revm/commit/69e302b48248d56dc285631b7f88332017bd4d0b#diff-8ef36919563c5922c56bf87d70a864582b8f2ddefe35e879542c208bbddc5484

I assumed that all new errors would be added at the end, like we do with our error codes in Hardhat, but I guess that's not the case. Good to know.

Reply:
(To be clear: you are right, is 0x65 now)

Reply:
So, to be clear: in the latest commit it's 0x65, so we should keep that in mind when we upgrade revm.

Reply:
Yup, I just ran into that problem myself when doing the `revm` update. Corrected is as you suggested. Thanks!